introduction_to_applying_machine_learning/deepar_chicago_traffic_violations/deepar_chicago_traffic_violations.ipynb

```diff
@@ -0,0 +1,467 @@
+    "\n",
+    "---\n",
+    "\n",
+    "---\n",
+    "\n",
+    "\n",
+    "## Contents\n",
+    "\n",
+    "1. [Background](#Background)\n",
+    "1. [Setup](#Setup)\n",
+    "1. [Data](#Data)\n",
+    "1. [Train](#Train)\n",
+    "1. [Host](#Host)\n",
+    "  1. [Evaluate](#Evaluate)\n",
+    "1. [Extensions](#Extensions)\n",
+    "\n",
+    "---\n",
+    "\n",
+    "## Background\n",
+    "\n",
+    "This notebook demonstrates time series forecasting using the Amazon SageMaker DeepAR algorithm by analyzing city of Chicago’s red light traffic camera violations. The dataset is hosted by [Data.gov](https://data.gov), and is managed by the [U.S. General Services Administration, Technology Transformation Service](http://www.gsa.gov/portal/category/25729).\n",
+    "\n",
+    "These violations are captured by camera systems and available to improve the lives of public from the [city of Chicago data portal](https://data.cityofchicago.org/). The [Speed Camera Violation  data](https://data.cityofchicago.org/Transportation/Red-Light-Camera-Violations/spqx-js37) set can be used to discern patterns in the data and gain meaningful insights.\n",
+    "\n",
+    "The data set contains multiple camera locations and daily violation counts. Each daily violations for a camera can be considered a separate time series. Amazon SageMaker’s DeepAR algorithm can be used to train a model for multiple streets simultaneously, and predict violation for multiple street cameras using the Amazon SageMaker’s [DeepAR algorithm](https://docs.aws.amazon.com/sagemaker/latest/dg/deepar.html).\n",
+    "\n",
+    "---\n",
+    "\n",
+    "## Setup\n",
+    "\n",
+    "This notebook was created and tested on an ml.m4.xlarge notebook instance.\n",
+    "\n",
+    "Let's start by specifying:\n",
+    "\n",
+    "- The S3 bucket and prefix that you want to use for training and model data.  This should be within the same region as the notebook instance, training, and hosting.\n",
+    "- The IAM role arn used to give training and hosting access to your data. See the documentation for how to create these.  Note, if more than one role is required for notebook instances, training, and/or hosting, please replace `sagemaker.get_execution_role()` with the appropriate full IAM role arn string(s)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sagemaker\n",
+    "\n",
+    "sess = sagemaker.Session()\n",
+    "bucket = sess.default_bucket()\n",
+    "base = 'DEMO-deepar-chicago' # notebook auther to input a short descriptive name\n",
+    "prefix = 'sagemaker/' + base\n",
+    "region = sess.boto_region_name\n",
+    "\n",
+    "role = sagemaker.get_execution_role()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we'll import the Python library like s3fs, matplotlib, pandas and numpy"
```

Comment:
* general comment: use present tense for describing what's happening in the code cells
* for this cell:
  * maybe word it as "Now we import some Python libraries used in this notebook, such as s3fs, matplotlib, pandas, and numpy"
  * also, can you add a sentence explaining the purpose of the `np.random.seed` call?

Reply:
fixed the tense, removed np.random.seed, no longer needed